lib/core/test/unit/Cardano/WalletSpec.hs

```diff
@@ -420,18 +418,11 @@ setupFixture (wid, wname, wstate) = do
 -- implements a fake signer that still produces sort of witnesses
 dummyTransactionLayer :: TransactionLayer DummyTarget ShelleyKey
 dummyTransactionLayer = TransactionLayer
-    { mkStdTx = \keyFrom inps outs -> do
+    { mkStdTx = \_keyFrom inps outs -> do
         let inps' = map (second coin) inps
         let tid = mkTxId inps' outs
         let tx = Tx tid inps' outs
-        wit <- forM inps $ \(_, TxOut addr _) -> do
-            (xprv, Passphrase pwd) <- withEither
-                (ErrKeyNotFoundForAddress addr) $ keyFrom addr
-            let (Hash sigData) = txId tx
-            let sig = CC.unXSignature $ CC.sign pwd (getKey xprv) sigData
-            return $ TxWitness
-                (CC.unXPub (getKey $ publicKey xprv) <> sig)
-        return (tx, wit)
+        return (tx, SignedTxBinary mempty)
```

Comment:
This makes the above test `walletKeyIsReencrypted` pointless! The dummy signer is essential to the good functioning of the test above. Please revert this change :+1: